internal/cli/atlas/serverless/backup/snapshots/list.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshots
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type ListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	cli.ListOpts
+	clusterName string
+	store       store.ServerlessSnapshotsLister
+}
+
+func (opts *ListOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var listTemplate = `ID	TYPE	STATUS	CREATED AT	EXPIRES AT{{range .Results}}
+{{.ID}}	{{.SnapshotType}}	{{.Status}}	{{.CreatedAt}}	{{.ExpiresAt}}{{end}}
+`
+
+func (opts *ListOpts) Run() error {
+	listOpts := opts.NewListOptions()
+	r, err := opts.store.ServerlessSnapshots(opts.ConfigProjectID(), opts.clusterName, listOpts)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas serverless backups snapshots list <clusterName> [--projectId projectId] [--page N] [--limit N].
+func ListBuilder() *cobra.Command {
+	opts := new(ListOpts)
+	cmd := &cobra.Command{
+		Use:     "list <clusterName>",
+		Short:   "Return all cloud backup snapshots for your project and serverless instance.",
+		Long:    fmt.Sprintf(usage.RequiredRole, "Project Read Only"),
+		Aliases: []string{"ls"},
+		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"clusterNameDesc": "Name of the Atlas serverless instance that contains the snapshots you want to retrieve.",
+		},
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of snapshots for the cluster named myDemo 
```

Comment:
```suggestion
		Example: fmt.Sprintf(`  # Return a JSON-formatted list of snapshots for the instance named myDemo 
```
I think we prefer instance when talking about serverless